apps/site/pages/components/molecules/breadcrumb.mdx

```diff
@@ -0,0 +1,229 @@
+---
+description: Breadcrumbs indicate the user's location in a website hierarchy.
+keywords:
+  - UI library
+  - ecommerce
+  - react
+sidebar_custom_props:
+  image: https://vtexhelp.vtexassets.com/assets/docs/src/Breadcrumb___20a8572e919e1262c1405a5e80009e25.png
+---
+
+import { Tab, Tabs } from 'nextra-theme-docs'
+import PropsSection from 'site/components/PropsSection'
+import { TokenTable, TokenRow, TokenDivider } from 'site/components/Tokens'
+import { OverviewSection } from 'site/components/OverviewSection'
+import { Breadcrumb } from '@faststore/ui'
+
+import { breadcrumbList } from '../../../mocks/breadcrumbList'
+
+<header>
+
+# Breadcrumb
+
+Breadcrumbs indicate the user's location in a website hierarchy.
+
+</header>
+
+## Overview
+
+<Tabs items={['Example', 'Code']} defaultIndex="0">
+  <Tab>
+    <OverviewSection>
+      <Breadcrumb breadcrumbList={breadcrumbList} />
+    </OverviewSection>
+  </Tab>
+  <Tab>
+    ```tsx
```

Comment:
I think we can show the breadcrumbList example here too.

Reply:
> I'm not sure, but something seems off, the links are collapsing, but I think when on the desktop they shouldn't (at least before it was not 🤔 )
> 
> <img alt="image" width="1043" src="https://user-images.githubusercontent.com/3356699/218722724-79b7d88a-b7a5-40f8-84b0-b768db257896.png">
> 
> Previous: <img alt="image" width="825" src="https://user-images.githubusercontent.com/3356699/218723394-8d78634b-b1ce-486a-8bc9-0e4b6eb18c84.png">

It happened because, in the desktop version, we have a limit of 50% for the breadcrumb, but in the document, this limit makes the breadcrumb small